wallet/src/de/schildbach/wallet/wallofcoins/api/RestApi.java

```diff
@@ -0,0 +1,114 @@
+
+import java.util.List;
+import java.util.Map;
+
+import de.schildbach.wallet.request.CreateAuthReq;
+import de.schildbach.wallet.wallofcoins.response.AdsListActivityResp;
+import de.schildbach.wallet.wallofcoins.response.CaptureHoldResp;
+import de.schildbach.wallet.wallofcoins.response.CheckAuthResp;
+import de.schildbach.wallet.wallofcoins.response.ConfirmDepositResp;
+import de.schildbach.wallet.wallofcoins.response.CreateAdResp;
+import de.schildbach.wallet.wallofcoins.response.CreateAuthResp;
+import de.schildbach.wallet.wallofcoins.response.CreateDeviceResp;
+import de.schildbach.wallet.wallofcoins.response.CreateHoldResp;
+import de.schildbach.wallet.wallofcoins.response.CurrentAuthResp;
+import de.schildbach.wallet.wallofcoins.response.DiscoveryInputsResp;
+import de.schildbach.wallet.wallofcoins.response.GetAuthTokenResp;
+import de.schildbach.wallet.wallofcoins.response.GetCurrencyResp;
+import de.schildbach.wallet.wallofcoins.response.GetOffersResp;
+import de.schildbach.wallet.wallofcoins.response.GetPricingOptionsResp;
+import de.schildbach.wallet.wallofcoins.response.GetReceivingOptionsResp;
+import de.schildbach.wallet.wallofcoins.response.OrderListResp;
+import de.schildbach.wallet.wallofcoins.response.SendVerificationResp;
+import de.schildbach.wallet.wallofcoins.response.VerifyAdResp;
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.DELETE;
+import retrofit2.http.Field;
+import retrofit2.http.FieldMap;
+import retrofit2.http.FormUrlEncoded;
+import retrofit2.http.GET;
+import retrofit2.http.POST;
+import retrofit2.http.Path;
+import retrofit2.http.Query;
+
+/**
+ * RestApi Client Interface For all WOC RestFull API with Method
+ * Get,Post,Update & Delete API call
+ */
+public interface RestApi {
+
+    @GET("api/v1/auth/current/")
+    Call<CurrentAuthResp> getCurrentAuth();
+
+    @POST("api/v1/auth/")
+    Call<CreateAuthResp> createAuth(@Body CreateAuthReq createAuthReq);
+
+    @GET("api/v1/orders/")
+    Call<List<OrderListResp>> getOrders(@Query("publisherId") String publisherId);
+
+    @GET("api/v1/holds/")
+    Call<List<OrderListResp>> getHolds();
+
+    @GET("api/v1/auth/{phone}/")
+    Call<CheckAuthResp> checkAuth(@Path("phone") String username, @Query("publisherId") String publisherId);
+
+    @DELETE("api/v1/auth/{phone}/")
+    Call<CheckAuthResp> deleteAuth(@Path("phone") String username, @Query("publisherId") String publisherId);
+
+    @DELETE("api/v1/orders/{orderId}/")
+    Call<Void> cancelOrder(@Path("orderId") String orderId, @Query("publisherId") String publisherId);
+
+    @FormUrlEncoded
+    @POST("api/v1/auth/{phone}/authorize/")
+    Call<GetAuthTokenResp> getAuthToken(@Path("phone") String username, @FieldMap Map<String, String> partMap);
+
+    @GET("api/v1/banks")
+    Call<List<GetReceivingOptionsResp>> getReceivingOptions(@Query("country") String country, @Query("publisherId") String publisherId);
+
+    @GET("api/v1/ad")
```

Comment:
must end the API call with a backslash

Reply:
@rgenito All API  are update base on with a backslash